docs/settings/alert-action-settings.asciidoc

```diff
@@ -41,6 +41,9 @@ You can configure the following settings in the `kibana.yml` file.
 [cols="2*<"]
 |===
 
+| `xpack.actions.enabled` {ess-icon}
```

Comment:
I don't think this is in the cloud allowlist. I don't see it in here: https://github.com/elastic/cloud/blob/master/scala-services/adminconsole/src/main/resources/settings/kibana/alerting.yml

Should we add it?

Reply:
Not sure about this config option at all :-) @mikecote what do you think?

Reply:
I have a feeling https://github.com/elastic/kibana/issues/90934 will go through for 8.0, which would reverse this work. I'm +1 to not allow-list it until someone asks, and when they do, we can gather their use case as to why.